lib/Slider/Slider.js

```diff
@@ -898,6 +898,7 @@ module.exports = kind(
 	*/
 	previous: function () {
 		this.set('value', this.value - this._jumpIncrementAmount);
+		if (this.popup && this.$.popup.getShowing()) return true;
```

Comment:
Hi @aarontam 
Currently popup is hided when we release mouse pointer pressing.
Do you mean popup should not hide?